packages/rfw/test/runtime_test.dart

```diff
@@ -347,6 +347,17 @@ void main() {
     expect(tester.widget<ColoredBox>(find.byType(ColoredBox)).color, const Color(0x00000002));
   });
 
+  testWidgets('Runtime', (WidgetTester tester) async {
+    final Runtime runtime = Runtime()
+          ..update(const LibraryName(<String>['core']), createCoreWidgets());
+    for (final MapEntry(key: LibraryName libName, value: WidgetLibrary lib) in runtime.libraries.entries) {
+      expect(libName.toString(), equals('core'));
+      if (lib case final LocalWidgetLibrary localWidgetLibrary) {
```

Comment:
I have no idea what this line of code does.

Reply:
this is a "pattern" it checks if "lib" is an instance of LocalWidgetLibrary and then creates a local variable named localWidgetLibrary that is available in the if block.

Reply:
it seems like `if (lib is LocalWidgetLibrary)` would be easier to understand?